external_parser/example_joiner.cc

```diff
@@ -466,6 +466,7 @@ int example_joiner::process_joined(v_array<example *> &examples) {
     // add an empty example to signal end-of-multiline
     examples.push_back(&VW::get_unused_example(_vw));
     _vw->example_parser->lbl_parser.default_label(&examples.back()->l);
+    examples.back()->is_newline = true;
```

Comment:
is this the right place?

Reply:
https://github.com/VowpalWabbit/vowpal_wabbit/pull/2890/files

Reply:
yup